woocommerce/woocommerce-ios
# Updates WordPressUI dependency

### Details:
This PR updates the WordPressUI dependency, and thus... shuts down the **Swift 4.2 Warning** we had for a while now.

(About time!!)

cc @bummytime @mindgraffiti Thank you!!


### Testing:
0. `pod install`
1. Build the app
2. Verify no warnings show up
